src/components/GobanLineSummary/GobanLineSummary.tsx

```diff
@@ -75,7 +100,13 @@ export class GobanLineSummary extends React.Component<GobanLineSummaryProps, any
             this.sync_state();
         });
 
-        this.goban.on("pause-text", (new_text) => this.setState({
+        // It appears "pause-text" is not part of the Goban event emitter.
```

Comment:
I'm 99% sure this pause text is not used anymore

Reply:
Cool cool I can remove it in a follow up PR :)